### DISCUSSION TITLE
Tying PyANSYS Geometry to PyMechanical

### DISCUSSION THREAD

**Discussion by jziadat** [ansys/pyansys#429] [Q&A] [answered]
Hello - I'm looking for an example where a geometry generated using PyANSYS Geometry is flowed into PyMechanical for analysis.  Can someone point me in the right direction?  Thanks!

**Comment by RobPasMue** [comment, marked as answer]
Hi @jziadat!

Once a geometry is generated using PyAnsys Geometry, you can export it into various formats that can then be read with the solver of your choice. In the case of PyMechanical, you might want to import it using PMDB.

The various formats supported by PyGeometry (when used with the Ansys Geometry Service) can be found here: https://geometry.docs.pyansys.com/version/0.4/api/ansys/geometry/core/designer/design/DesignFileFormat.html#designfileformat

An example of how to export your model can be found here: https://geometry.docs.pyansys.com/version/0.4/examples/01_getting_started/04_modeling.html#Export-files

Hope it helps!

**Comment by jziadat** [comment]
Thanks!
